taxonomies/categories.txt

```diff
@@ -34541,8 +34544,25 @@ zh:猪肉餐
 <en:Pork meals
 <en:skewers
 en:pork skewers
-fr:Brochettes de porc, brochette de porc
-nl:Varkensspiesjes
+fr:Brochettes de porc, brochette de porc, échine de porc rôtie, échine de porc au four
```

Comment:
Here we want only synonyms, and "échine de porc au four" is not a synonym for "brochettes de porc".

Reply:
Yes, you right, thanks